src/plonesocial/microblog/statusupdate.py

```diff
@@ -62,6 +64,14 @@ def _init_context(self, context):
             # actual object context
             self.context_object = context
 
+    def _init_mentions(self, mention_ids):
+        self.mentions = {}
+        if mention_ids is None:
+            return
+        for userid in mention_ids:
+            user = api.user.get(userid)
```

Comment:
As api.user.get() returns None on an invalid userid, I have changed this to simply ignore invalid userids.